app/models/comment.rb

```diff
@@ -0,0 +1,12 @@
+class Comment < ApplicationRecord
+  belongs_to :user, optional: true
+  belongs_to :post, optional: true
+
+  after_save :update_comment_counter
+
+  private
+
+  def update_comment_counter
+    Post.find(post_id).update(comments_counter: Comment.where(post_id:).count)
+  end
```

Comment:
- [ ] Refer to my coment on using `Increment!` method like so 👇🏾 

```ruby
def update_comments_counter
  post.increment!(:comments_counter)
end
```

Reply:
Fixed.